updating/update-using-custom-machine-config-pools.adoc

```diff
@@ -0,0 +1,76 @@
+
+Using an IPSU can also help with large clusters with hundreds of nodes that cannot be fully updated in your organization's defined maintenance window, or if your organization allows multiple maintenance windows. The IPSU process is *not* a typical update workflow. With larger clusters, it can be a time-consuming process that requires you execute multiple commands. This complexity can result in errors that can affect the entire cluster. It is recommended that you carefully consider whether your organization want to use an IPSU and carefully plan implementing the process before you start.
+
+[NOTE]
+====
+It is not recommended to update the MCPs to different {product-title} versions. For example, do not update one MCP from 4.y.10 to 4.y+1.10 and another to 4.y+1.20.
+This scenario has not beeb tested and might result in an undefined cluster state.
+====
+
+The IPSU process described in this topic involves: 
+
+* creating one or more custom machine config pools (MCPs) 
+* labeling each node that you do not want to  upgrade immediately to move those nodes to the custom MCPs 
+* pausing those custom MCPs, which prevents updates to those nodes
+* performing the cluster update
+* removing the custom labels to move some nodes back to their original MCP which triggers the update on those nodes
+* testing the applications on those nodes to make sure the applications work as expected on those newly-updated nodes
+* removing the custom labels from the remaining nodes in small batches and testing the applications on those nodes 
+
+//The following wording comes from https://github.com/openshift/openshift-docs/pull/34704, not yet finalized
+
+[NOTE]
+====
+Pausing a machine config pool prevents the Machine Config Operator from applying any configuration changes on the associated nodes. Pausing an MCP also prevents any automatically-rotated certificates from being pushed to the associated nodes, including the automatic CA rotation of the `kube-apiserver-to-kubelet-signer` CA certificate. If the MCP is paused when the `kube-apiserver-to-kubelet-signer` CA certificate expires and the MCO attempts to automatially renew the certificate, the new certificate is created but not applied across the nodes in the respective machine config pool. This causes failure in multiple `oc` commands, including but not limited to `oc debug`, `oc logs`, `oc exec`, and `oc attach`. Pausing an MCP should be done with careful consideration about the `kube-apiserver-to-kubelet-signer` CA certificate expiration and for short periods of time only.  
+====
+
+To ensure that the cluster remains stable, creating a custom MCP for the control plane nodes (also known as the master nodes) is not supported. The Machine Config Operator (MCO) ignores any custom MCP created for the control plane nodes.
```

Comment:
This should be a note IMO.

Reply:
I can do that. I just dislike _stacking_ multiple notes together. Although, it is quite common in our docs.